www/caddy/src/opnsense/mvc/app/controllers/OPNsense/Caddy/forms/dialogHandle.xml

```diff
@@ -60,28 +60,32 @@
         <style>tokenize</style>
         <allownew>true</allownew>
         <hint>192.168.1.1</hint>
-        <help><![CDATA[Enter the internal domain name or IP address of the upstream destination for this handler. If multiple upstream destinations are chosen, they will be load balanced with the default random policy.]]></help>
+        <help><![CDATA[Enter a domain name or IP address of the upstream destination. If multiple are chosen, they will be load balanced with the default random policy. A health check can be set in "Load Balancing"]]></help>
     </field>
     <field>
         <id>handle.ToPort</id>
         <label>Upstream Port</label>
         <type>text</type>
         <hint>80</hint>
-        <help><![CDATA[Enter the port number of the upstream destination. Leave this empty to use port 80.]]></help>
+        <help><![CDATA[Leave empty to use port 80 or choose a custom port for the upstream destination.]]></help>
```

Comment:
```suggestion
        <help><![CDATA[Leave empty to use the default port or choose a custom port for the upstream destination.]]></help>
```

As the default is shown as a hint we better not hardcode it here as well.

Reply:
Yes that makes sense. Thank you.